gui.py

```diff
@@ -1632,7 +1632,10 @@ def update_notifications(self) -> None:
         self._settings.tray_notifications = bool(self._vars["tray_notifications"].get())
 
     def _get_autostart_path(self, tray: bool) -> str:
-        self_path = f'"{SELF_PATH.resolve()!s}"'
+        if IS_APPIMAGE:
+            self_path = f'"{APPIMAGE_PATH!s}"'
+        else:
+            self_path = f'"{SELF_PATH.resolve()!s}"'
```

Comment:
```suggestion
        self_path = f'"{SELF_PATH.resolve()!s}"'
```

Reverts the change. `SELF_PATH` points to the `.AppImage` file after all.